Hymns/ContentView.swift

```diff
@@ -9,11 +9,25 @@
 import SwiftUI
 
 struct ContentView: View {
+    var hymns = hymnTestData
+    @State private var searchText : String = ""
+    
+    
     var body: some View {
-        Text("Hello, World!")
+                       NavigationView {
+                            VStack {
+                                SearchBar(text: $searchText)
+                                List {
+                                    ForEach(hymns.filter { self.searchText.isEmpty ? true : $0.contains(self.searchText)}, id: \.self) { hymn in
+                                        Text(hymn)
+                                }
+                                }.navigationBarTitle(Text("Look up any hymn"))
+                            }
+                        }
```

Comment:
nit: indenting looks funny. could you fix?

Reply:
Done, and I learned a valuable lesson here. When you make file changes like I did adding two files then you have to submit the xcodeproj file to github. Otherwise xcode does not pick the files up when you pull them into your local. So always submit xcodeproj when adding or deleting files.